### PR TITLE
Fix name of the config_src variable in neutron_sriov role

### PR DESCRIPTION
**Pull request by slawqo**

### CODE REVIEW

**Review by jpodivin — CHANGES_REQUESTED** [review]
Please add the new variable into argspec.

**Review by booxter — CHANGES_REQUESTED** [review]

File: roles/edpm_neutron_sriov/tasks/configure.yml [hunk 1]
```diff
@@ -32,9 +32,9 @@
       notify:
       - restart neutron-sriov-agent
 
-    - name: discover secrets in {{ edpm_ovn_config_src }}
+    - name: discover secrets in {{ edpm_neutron_sriov_config_src }}
```
[the remark this answers is not in the document]
Comment (slawqo) [reply roles/edpm_neutron_sriov/tasks/configure.yml]: Done

**Comment by slawqo** [comment]
> Please add the new variable into argspec.

done

**Comment by booxter** [comment]
```
        "TASK [edpm_neutron_sriov : discover secrets in /var/lib/openstack/configs/neutron-sriov] ***",
Warning: [WARNING]: Skipped '/var/lib/openstack/configs/neutron-sriov' path due to this",
        "access issue: '/var/lib/openstack/configs/neutron-sriov' is not a directory",
        "ok: [fake-sriov-compute-1 -> localhost]",
```

**Comment by slawqo** [comment]
recheck

> ```
>         "TASK [edpm_neutron_sriov : discover secrets in /var/lib/openstack/configs/neutron-sriov] ***",
> Warning: [WARNING]: Skipped '/var/lib/openstack/configs/neutron-sriov' path due to this",
>         "access issue: '/var/lib/openstack/configs/neutron-sriov' is not a directory",
>         "ok: [fake-sriov-compute-1 -> localhost]",
> ```

isn't it like that because there is no actually /var/lib/openstack/configs/neutron-sriov there during the molecule test?

**Comment by booxter** [comment]
OK I know the reason why your config file with transport_url was not properly loaded. You need this fix to point molecule to the directory with the config that mimics the one from Secret prepared by neutron-operator:

```
diff --git a/roles/edpm_neutron_sriov/molecule/default/converge.yml b/roles/edpm_neutron_sriov/molecule/default/converge.yml
index 08d27ef..5746332 100644
--- a/roles/edpm_neutron_sriov/molecule/default/converge.yml
+++ b/roles/edpm_neutron_sriov/molecule/default/converge.yml
@@ -19,4 +19,4 @@
   roles:
     - role: "edpm_neutron_sriov"
       vars:
-        edpm_ovn_config_src: "{{lookup('env', 'MOLECULE_SCENARIO_DIRECTORY')}}/test-data"
+        edpm_neutron_sriov_agent_config_src: "{{lookup('env', 'MOLECULE_SCENARIO_DIRECTORY')}}/test-data"
```

With that, you should be able to:

1. revert the patch that moves log-file from config file to command.
2. revert the switch from neutron.conf.d to /etc/neutron/conf.d path.

**Review by booxter — CHANGES_REQUESTED** [review]
Please fix roles/edpm_neutron_sriov/molecule/default/converge.yml to point to the right test-data path. Then revert the switch to /etc/neutron/conf.d and the move of the log-dir.

**Comment by slawqo** [comment]
> Please fix roles/edpm_neutron_sriov/molecule/default/converge.yml to point to the right test-data path. Then revert the switch to /etc/neutron/conf.d and the move of the log-dir.

Done, thx a lot :)

**Comment by booxter** [comment]
```
error: unable to default to a user name: the server is currently unable to handle the request (get users.user.openshift.io ~)
```

**Comment by booxter** [comment]
recheck

**Comment by slawqo** [comment]
recheck

**Review by karelyatin — APPROVED** [review]
/lgtm

**Comment by Sandeepyadav93** [comment]
recheck

**Comment by slawqo** [comment]
recheck

**Comment by booxter** [comment]
recheck

**Pull request merged**